commands/deploy_test.go

```diff
@@ -115,3 +117,51 @@ func Test_getRegistryAuth_NotRequiredForLocalImage(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func Test_deployFailed(t *testing.T) {
+
+	var failedDeploy = make(map[string]int)
+	var containedErrorsCount int
+	failedDeploy["example1"] = 100
+	failedDeploy["example2"] = 300
+	failedDeploy["example3"] = 400
+	failedDeploy["example4"] = 500
+	err := deployFailed(failedDeploy)
+	if err == nil {
+		t.Errorf("\nHad to exit with errors!")
+		t.Fail()
+	}
+	for _, theErrorCode := range failedDeploy {
+		if strings.Contains(err.Error(), strconv.Itoa(theErrorCode)) {
+			containedErrorsCount++
+		}
+	}
+	if containedErrorsCount != len(failedDeploy) {
+		t.Errorf("\nWanted: %d number of erros and got: %d!", len(failedDeploy), containedErrorsCount)
+		t.Fail()
+	}
+}
+func Test_deploySucceeded(t *testing.T) {
+	var succededDeploy = make(map[string]int)
+	if err := deployFailed(succededDeploy); err != nil {
+		t.Errorf("\nHad to exit with no errors!")
+		t.Fail()
+	}
+}
+func Test_badStatusCOde(t *testing.T) {
+	statusCode := 200
+	if badStatusCode(statusCode) {
+		t.Errorf("\nHad to be good status code!")
+		t.Fail()
+	}
+	statusCode = 202
+	if badStatusCode(statusCode) {
+		t.Errorf("\nHad to be good status code!")
+		t.Fail()
+	}
+	statusCode = 300
+	if !(badStatusCode(statusCode)) {
+		t.Errorf("\nHad to be bad status code!")
```

Comment:
This error won't help track down the issue in a build log. Please use: `t.Errorf("\nStatus code unexpected - want: %d, but got %d.")` or similar